prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+
+    }
+
+    .row {
+      display: flex;
+      width: 100%;
+      flex-basis: 25%
+    }
+
+    .column {
+      flex-basis: calc(100%/3);
+      padding-right: 8px;
+      padding-left: 8px;
+  }
+
+    .src{
+      display; flex;
+    width: 100%;
+  }
+
+  </style>
+</head>
+<body>
+  <h1>Indiana Jones</h1>
+  <div class="container">
+    <div class="row">
+      <div class="column">
+        <img src="images/raiders.jpg">
+        <h3>Raiders of the lost Ark</h3>
+        <p>Epic tale in which an intrepid archaeologist tries to beat a band of Nazis to a unique religious relic which is central
+        to their plans for world domination. Battling against a snake phobia and a vengeful ex-girlfriend, Indiana Jones is in
+        constant peril, making hair's-breadth escapes at every turn in this celebration of the innocent adventure movies of an
+        earlier era.</p>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="column">
```

Comment:
Hi Amir, to get your columns lining up properly, you'll want to use only one container and one row. All three columns will go directly inside of the row.